### ISSUE TITLE
flatpak vs rpm

### ISSUE THREAD

**Issue by busterbeam** [ddnexus/Flat-Plat-Equilux#36]
I noticed some of my apps didn't have the theme applied and they defaulted back to adwaita(-white) which obvoiusly is not nice.  I think it's apps that use flatpak instead of rpm (which I prefer).  You don't need to fix this but I would advise adding this to your readme.

**Comment by ajz3d** [comment]
I know your pain, but until someone packages our favorite theme and uploads it to Flathub, we will have to live with it.

You can however enforce *Adwaita-Dark* theme in Flatpak programs (it's mostly about the Qt ones). There's a lot of information about this on [ArchWiki](https://wiki.archlinux.org/index.php/Uniform_look_for_Qt_and_GTK_applications#Themes_not_working_in_GTK_apps), but it basically boils down to installing `org.kde.KStyle.Adwaita`, `org.gtk.Gtk3theme.Adwaita-dark` from Flathub and setting:
```
QT_STYLE_OVERRIDE=adwaita-dark
QT_QPA_PLATFORMTHEME=gtk2
```
in your `.xsessionrc` file.

Also, install `qt5-styleplugins` package from your distro repository.

Adwaita-dark is not Equilux, but at least it won't burn out your eyes. You could also try with some darker variety of *Materia* theme, which Equilux is a fork of. I see that it's available on Flathub:
```
Materia-compact GTK theme	org.gtk.Gtk3theme.Materia-compact		3.22
Materia-dark-compact GTK theme	org.gtk.Gtk3theme.Materia-dark-compact		3.22
Materia-dark GTK theme	org.gtk.Gtk3theme.Materia-dark		3.22
Materia-light-compact GTK theme	org.gtk.Gtk3theme.Materia-light-compact		3.22
Materia-light GTK theme	org.gtk.Gtk3theme.Materia-light		3.22
Materia GTK theme	org.gtk.Gtk3theme.Materia		3.22
```
Though I don't see a Qt version.

**Comment by busterbeam** [comment]
Do you plan to make a light version aswell?  As I use gnome-shell-extension night-theme-switcher.  And I would prefer it to be equalix "light" during the day and dark during the night.  I believe that is the best way to mitgate disrupting your rythm

**Comment by ajz3d** [comment]
There's no Equilux "light" as far as I'm aware, and judging by the date of the last commit (Oct 29, 2018), this repository is most likely abandoned, even though it's not marked as archived, so you shouldn't expect new features. At least not until something breaks.

What you could do though, is to check the forks and find the most active one. Maybe someone can be persuaded to work on a light version of this theme.

**Comment by ddnexus** [comment]
@ajz3d you are right... the theme is mostly abandoned because it would require too much time to keep it up do date with the upstream theme... besides there are a few other factors that contribute to the problem.

It's worth noting that vanilla upstream removed the ugly color cast from the background colors, so it's neutral now, although the contrast is still too high IMO. There are also other neutral themes that might be used as an alternative. 

Sorry... I never archived it because I always hope to have some time to dedicate to it again, but it doesn't look it will happen anytime soon.

**Comment by busterbeam** [comment]
I think I have found another bug that is trully annoying, in some apps, the icon's highlight on being clicked but the circle stays highlighted and it only disappears when you exit that window and open it again

**Comment by ajz3d** [comment]
@ddnexus
Don't worry. For the time being Equilux works without major problems (at least on XFCE 4.16) and hopefully it will stay this way for some time.

I tried Materia Dark, but like you say, its contrast ratio is way too hard for eyes. In fact the majority of dark themes seem to suffer from two common problems: they're either blue-tinted or have too much contrast. And both of those issues not only cause eye strain, but also affect color perception. Equilux is the only one I could find that has a perfect balance between light and dark gray shades. Thanks for making it.

> I think I have found another bug (...)

@busterbeam
I think you meant "a bug", because the problem you mentioned in your initial post is most definitely not a bug, but just the way how things work in a sandboxed environment like Flatpak. If you want the theme to be available in Flatpak, it must be packaged for Flatpak.

But yes, sometimes odd bugs happen. I didn't encounter this particular icon bug that you mention, but from time to time I see those... big bubbles indicating the end of a horizontal or vertical scrolling of file lists (e.g. in Thunar), staying visible until I close the program window. I learned not to notice them though.
